doc/manual/memory.rst

```diff
@@ -0,0 +1,46 @@
+
+.. contents:: Contents
+   :local:
+   :depth: 3
+
+Memory-efficient IO
+###################
+
+Preloading
+==========
+
+Raw
+^^^
+MNE-Python can read data on-demand using the ``preload`` option provided in :ref:`IO functions <ch_convert>`. For example::
+
+    from mne import io
+    from mne.datasets import sample
+    data_path = sample.data_path()
+    raw_fname = data_path + '/MEG/sample/sample_audvis_filt-0-40_raw.fif'
+    raw = io.Raw(raw_fname, preload=False)
+
+By default, most IO functions load data in chunks of approximately ``100 MB``.
```

Comment:
And actually I don't think it's true for *most* readers, and even for the ones it is true for, isn't true under all circumstances.